### PR TITLE
disable localClientDatabases, enable allowDataAccessInUserLanguage on gratisdatawiki

### PR DESCRIPTION
**Pull request by ugochimobi**

### CODE REVIEW

File: Wikibase.php [hunk 1]
```diff
@@ -269,6 +264,8 @@
 	$wgWBRepoSettings['useKartographerGlobeCoordinateFormatter'] = true;
 	// Data-Bridge
 	$wgWBRepoSettings['dataBridgeEnabled'] = true;
+	$wgWBRepoSettings['allowDataAccessInUserLanguage'] = true;
+	$wgWBRepoSettings['entityAccessLimit'] = '500';
```
Comment (Universal-Omega) [inline Wikibase.php:268]: Should this be an integer?

Comment (ugochimobi) [reply Wikibase.php:268]: > Should this be an integer?

Yeah, it is, but I probably didn't insert it correctly? but it's an integer.

Comment (Universal-Omega) [reply Wikibase.php:268]: Remove the quotations, otherwise it is a string.

Comment (ugochimobi) [reply Wikibase.php]: > Remove the quotations, otherwise it is a string.

ah, done, thanks.

**Pull request merged**